bitcoin/src/bip32.rs

```diff
@@ -583,19 +583,21 @@ impl Xpriv {
             .expect("BIP32 internal private key representation is broken")
     }
 
-    /// Attempts to derive an extended private key from a path.
+    /// Derive an extended private key from a path.
```

Comment:
We don't use the imperative tense for function docs. FYI when we discussed and started implementing this it was the correct choice, to the best of our knowledge, even though there are many crates that do use the imperative. Here is an arbitrary example from the std lib: 

https://doc.rust-lang.org/std/string/struct.String.html#method.reserve

I'm not super good with grammar, please correct me if you catch me making any mistakes, second language English speakers often have better grammar than first language English speakers in my experience, well often better than me anyway. I believe the tense we use is third person, where the function is the third person.

Reply:
I wouldn't normally pick this up in review but since you are interested in docs I thought it worth mentioning.

Reply:
Really helpful thanks! Yes, I am interested in docs and technical documentation.
I agree with this. Will update it.